### ISSUE TITLE
Add a bookmarklet to apply this to any site

### ISSUE THREAD

**Issue by waldyrious** [waldyrious/unstyle#19]
Similar to Readability or Instapaper

**Comment by waldyrious** [comment]
I'll keep updating the one I'm currently using locally for tests:

```js
javascript:document.body.removeAttribute('background');
for(ss of document.styleSheets){void(ss.disabled=true)}
for(el of document.getElementsByTagName('*')){void(el.style.cssText='');void(el.removeAttribute('bgColor'));}
for(t of document.getElementsByTagName('table')){t.border=0}
for(f of document.getElementsByTagName('font')){f.removeAttribute('size');f.removeAttribute('face');f.removeAttribute('color')}
document.head.insertAdjacentHTML('beforeend',
    '<link rel="stylesheet" href="'+
    (window.location.protocol.startsWith('http')?window.location.protocol:'http:')+
    '//rawgit.com/waldyrious/default.css/master/default.css"/>'
)
```

**Comment by waldyrious** [comment]
This has been implemented in 951fa32 :)
